app/jobs/compute_auxiliary_data.rb

```diff
@@ -10,4 +10,138 @@ def self.reason_not_to_run
   def perform
     AuxiliaryDataComputation.compute_everything
   end
+
+  def recompute_popular_rankings
```

Comment:
I'm not seeing how this is called/invoked - AuxiliaryDataComputation doesn't invoke it, and I don't see any hooks to run it after CAD has been completed. Grepping for the method name only yields the method definition

Reply:
Uhm... Look again please? :innocent: 

(Yes, this was my fault and I just pushed a commit to fix it...)